scripts/generate-docs/BeautifierDoc.ts

```diff
@@ -81,46 +86,118 @@ export default class BeautifierDoc extends Doc {
+      return builder;
+    }
+    builder.header("Prerequisites", 2);
+    builder.append("Please install the following prerequisites:");
+    const depLinks = dependencies.map((dep: ExecutableDependencyDefinition) =>
+      MarkdownBuilder.createDocLink(
+        dep.name,
+        `executable-${slugify(this.beautifier.name)}-${slugify(dep.name)}`,
+      ),
+    );
+    builder.list(depLinks);
+    return builder;
+  }
+
   private get packagePeerDependencies(): string[] {
     return Object.keys(_.get(this.pkg, "peerDependencies", {}));
   }
+
   private appendUsageSection(builder: MarkdownBuilder): MarkdownBuilder {
     builder.header("Usage", 2);
+
+    const beautifierName: string = this.beautifier.name;
+
     builder.append(
-      "Below are example configuration files for each of the supported languages."
+      `Enable ${beautifierName} beautifier for a language by adding the beautifier's name, \`${beautifierName}\`, to \`beautifiers\` option array.`,
     );
-    const beautifierName: string = this.beautifier.name;
-    this.languages.forEach(lang => {
-      builder.details(lang.name, builder => {
-        builder.append(
-          `A \`.unibeautifyrc.json\` file would look like the following:`
-        );
-        builder.code(
-          JSON.stringify(
-            {
-              [lang.name]: {
-                beautifiers: [beautifierName],
-              },
-            },
-            null,
-            2
-          ),
-          "json"
-        );
-      });
-    });
+
+    const { dependencies } = this;
+    if (dependencies.length === 0) {
+      return builder;
+    }
+    const executableConfig = dependencies
+      .filter(dep => dep.type === DependencyType.Executable)
+      .reduce(
+        (config, dep: ExecutableDependencyDefinition) => ({
+          ...config,
+          [dep.name]: {
+            path: fakePathForExecutable(dep),
+          },
+        }),
+        {},
+      );
+
+    const beautifierOptions: any = {
+      ...executableConfig,
+    };
+    const canResolveConfig = !!this.beautifier.resolveConfig;
+    if (canResolveConfig) {
+      beautifierOptions.prefer_beautifier_config = true;
+    }
+
+    builder.append("");
+    builder.append(
+      `A \`.unibeautifyrc.json\` file would look like the following, including some more advanced options:`,
+    );
+    builder.code(
+      JSON.stringify(
+        {
+          LANGUAGE_NAME: {
+            beautifiers: [beautifierName],
+            [beautifierName]: beautifierOptions,
+          },
+        },
+        null,
+        2,
+      ),
+      "json",
+    );
+    builder.append(
+      `**Note**: The \`LANGUAGE_NAME\` should be replaced with your desired supported language name, such as ${this.languages
+        .slice(0, 3)
+        .map(lang => `\`${lang.name}\``)
+        .join(", ")}, etc.`,
+    );
+
+    if (canResolveConfig) {
+      builder.header("Prefer Beautifier Specific Configuration File", 3);
```

Comment:
Put some blank space above this h3.  Or make the h3's stand out a bit more.  This particular one is hard to see that it's actually a heading.